add-new-chain-to-givEconomy-guide.md

```diff
@@ -0,0 +1,136 @@
+for other chains you should change them based on the chain
+
+```
+cardona = { key = "${ETHERSCAN_API_KEY}", url = "https://api-cardona-zkevm.polygonscan.com/api", chain = 2442 }
+```
+
+8. Enter this command to check everything is working locally
+
+`forge script script/deployTokenDistro.s.sol  --rpc-url  {rpcUrl}  --legacy --chain 2442 {Desired chainId}`
+
+
+9. Enter this command to broadcast and verify the smart contracts on blockchain
+
+`forge script script/deployTokenDistro.s.sol  --rpc-url  {rpcUrl}    --verify --broadcast --legacy --chain 2442 {Desired chainId}`
+
+9. Verify contract with proxy admin in the block explorer
+
+This is example of published `tokenDistro`: https://cardona-zkevm.polygonscan.com/address/0x2df3e67be4e441cddd2d29c3d41dfd7d516f18e6
+
+## Mint GIV test token for token distro
+We go to [token page in the block explorer](https://cardona-zkevm.polygonscan.com/address/0xa77390562986f5d08f5aecf5d3fb82bd16b44548#writeContract) and mint as much as token that we need for token distro address
+
+## Deploy Unipool givPower
+1. change some parms in below file
+https://github.com/Giveth/giveconomy-foundry/blob/develop/script/deployUnipoolGIVpower.s.sol
+
+```
+    address givTokenAddressOptimismSepolia = 0x2f2c819210191750F2E11F7CfC5664a0eB4fd5e6;
+    // address tokenDistroOptimismMainnet = 0xE3Ac7b3e6B4065f4765d76fDC215606483BF3bD1;
+    address tokenDistroOptimismSepolia = 0x301C739CF6bfb6B47A74878BdEB13f92F13Ae5E7;
+    ProxyAdmin unipoolGIVpowerProxyAdmin = ProxyAdmin(address(0x3b197F5cDa3516bD49e193df6F1273f3f16d414a));
+```
+all of these params are achieved in the previous steps
+
+2. Deploy it with foundry
+`forge script script/deployUnipoolGIVpower.s.sol  --rpc-url  {rpcUrl}   --legacy --chain 2442 --verify --broadcast`
+
+3. Verify contract with proxy admin in the block explorer
+
+4. We call `notifyRewardAmount` method and set some reward for the contract https://cardona-zkevm.polygonscan.com/address/0x7e9f30a74fcdf035018bc007f9930aa171863e33#writeProxyContract the `rewardDistributor` address can call it (on unipool smart contract)
+
+5. Call `transferOwnership` if it's needed
+
+6. Call `setRewardDistribution` on Unipool GivPower to set the wallet address which will call the `notifyRewardAmount`
+
+7. Check the address of `DISTRIBUTOR_ROLE` in tokenDistro contract https://cardona-zkevm.polygonscan.com/address/0x2df3e67be4e441cddd2d29c3d41dfd7d516f18e6#readProxyContract
+
+8. Call `grantRole` in `tokenDistro` https://cardona-zkevm.polygonscan.com/address/0x2df3e67be4e441cddd2d29c3d41dfd7d516f18e6#writeProxyContract and grant `DISTRIBUTOR_ROLE` to givPower unipool address
+
+9. Call `assign()` method in tokenDistro and assign the reward token amount to unipool givPower
+
+
+## Test
+1. Call `stake` method in unipoolGivPower contract
+2. See the giviverse expansion percentage in https://giveth.io/givstream
```

Comment:
giviverse?!

Reply:
<img width="961" alt="Screenshot 1403-07-16 at 1 12 16 in the afternoon" src="https://github.com/user-attachments/assets/23e7d63c-f659-47e4-8775-b593e30ce3d4">
Yes